src/main.js

```diff
@@ -49,7 +49,7 @@ if (process.env.NODE_ENV === "test") {
 
         routes() {
             this.timing = 0;
-            this.urlPrefix = 'http://localhost:5000';
+            this.urlPrefix = 'https://api.biascorrect.org/';
```

Comment:
Same goes for here :)
```suggestion
            this.urlPrefix = 'https://api.biascorrect.org';
```